charts/gardener/controlplane/values.yaml

```diff
@@ -1,7 +1,10 @@
+# NOTE: When making changes to default values & validation, please adjust the API comments and validation
+# in package landscaper/pkg/controlplane/apis/imports
 global:
   # Gardener API server configuration values
   apiserver:
     enabled: true
+    clusterIdentity: garden-cluster-identity
```

Comment:
This value was consciously not added to the `values.yaml` to force operators to provide a dedicated name. Otherwise, it can easily lead to conflicts and strange behaviour. Please remove it again.

Reply:
okay I see, will do.

Reply:
I would propose to generate a random gardener identity in the controlplane component if not explicitly provided (making deployment easy).
I will remove the value from the helm chart though as suggested.